src/app/tour/[id]/components/CommentCard/CommentCard.tsx

```diff
@@ -0,0 +1,23 @@
+import { ReactElement } from "react";
+import { MockComment } from "@/mocks/mockComments";
+import styles from "./CommentCard.module.css";
+import Image from "next/image";
+
+export default function CommentCard({
+  comment,
+}: {
+  comment: MockComment;
+}): ReactElement {
+  return (
+    <div className={styles.comment}>
+      <div className={styles["comment-header"]}>
+        <Image src={"/avatar.svg"} alt={""} width={50} height={50} />
```

Comment:
وقتی String دارید نیازید به آکلاد نیست

```suggestion
        <Image src="/avatar.svg" alt="" width={50} height={50} />
```